docker-compose.yml

```diff
@@ -19,14 +19,22 @@
 
 version: '3.5'
 
+# TODO(kszucs): set arrow's mount to :ro mode, once all of the builds are
```

Comment:
@wesm I had to disable the read-only mount, because it has caused tiny, but really annoying issues.

Reply:
No problem :)